pb/protos/view.proto

```diff
@@ -5,14 +5,81 @@ import "google/protobuf/any.proto";
 import "google/protobuf/timestamp.proto";
```

Comment:
And then in here, you can see all the other, non-model objects used throughout textile: